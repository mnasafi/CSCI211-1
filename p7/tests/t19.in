echo t19: Insert days of week into tree, then call balanced
insert Friday
insert Monday
insert Saturday
insert Sunday
insert Wednesday
insert Thursday
insert Tuesday
balanced
