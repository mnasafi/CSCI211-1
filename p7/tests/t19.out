t19: Insert days of week into tree, then call balanced
Tree is not balanced.
